include/myst/kernel.h

```diff
@@ -11,6 +11,14 @@
 #include <myst/uid_gid.h>
 #include <signal.h>
 
+typedef enum
+{
+    myst_fork_none = 0,
+    myst_fork_pseudo_wait_for_children,
+    myst_fork_pseudo_kill_children,
+    myst_fork_pseudo_wait_for_exit_exec
```

Comment:
Will you need ``myst_fork_wait_for_exec`` (where the parent waits until the ``exec()`` has started?

Reply:
that is the next PR